status-im/status-react
# [FEATURE] Encrypt realms

Fixes #3798 

### Summary:
- Adds Realm encryption with key stored in device keychain, using [react-native-keychain](https://github.com/oblador/react-native-keychain)
- Refactors app initialize flow to get or create device encryption key, then uses for each realm.open call

### Review notes:
- Need ability to delete or reset key in certain cases?
- Unclear how migrating from non-encrypted to encrypted will work

### Steps to test:
- Open Status
- Login, logout, create accounts, restart app, etc. to ensure no breaking changes

Status: Ready
